src/sass/partials/base/_content.scss

```diff
@@ -0,0 +1,70 @@
+@import 'content/variables';
+@import 'content/mixins';
+@import 'content/extends';
+
+* {
+  margin: 0;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+body {
+  background-color: $white;
+  font-family: $font-stack;
+  margin: 0;
+}
+
+h1,
+h2 {
+  padding: .5em 0;
+}
+
+.wrapper {
+
+  @include breakpoint($small) {
+    max-width: 600px;
+  }
+
+  @include breakpoint($medium) {
+    max-width: 900px;
+  }
+
+  @include breakpoint($large) {
+    max-width: 1200px;
+  }
+
+  margin: 0 auto;
+  margin-bottom: -50px;
+  margin-top: 50px;
+  min-height: 100%;
+  padding: 0 1em;
+
+  &:after {
+    content: '';
+    display: block;
+    height: 75px;
+  }
+
+  a {
+    color: color('teal', 60);
+    text-decoration: none;
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+
+  .punchcard-logomark {
+    width: 250px;
+    margin: 0 auto;
+    display: block;
+    margin-top: 75px;
+  }
+
+  ul.content-type-list {
```

Comment:
Please don't over-qualify selectors. This is `.wrapper ul.content-type-list`. It probably should only be a single class. Please also break this up in to components instead of a series of one-off classes